src/yasolr_config.cpp

```diff
@@ -124,6 +124,11 @@ void yasolr_init_config() {
   config.configure(KEY_WIFI_PASSWORD);
   config.configure(KEY_WIFI_SSID);
 
+  // Modbus Configuration
+  config.configure(KEY_ENABLE_VICTRON_MB, YASOLR_FALSE);
+  config.configure(KEY_VICTRON_MB_SERVER);
+  config.configure(KEY_VICTRON_MB_PORT);
```

Comment:
`config.configure(KEY_MODBUS_PORT, "502");` was ok.
only the server had to be without any default